pkgs/development/python-modules/atenpdu/default.nix

```diff
@@ -20,11 +20,16 @@ buildPythonPackage rec {
     hash = "sha256-KzRoE4tE/tQkKYroq5PbWKREmEl8AwbIOg3IHRZZtsQ=";
   };
 
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace-fail pysnmp-lextudio pysnmp
```

Comment:
The alternative would be `mkPythonMetaPackage` but maybe this is temporary?

Reply:
No energy for such a refactor right now.